user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css

```diff
@@ -1761,4 +1761,48 @@ hr {
   display: none;
 }
 
+.pentaho-tab-bar{
+  display: flex;
+  width: 100vw;
+  overflow-x: auto;
+  scrollbar-width: thin;
+  overflow-y: hidden;
+}
+
+:where(.burger-mode) .pentaho-tab-bar{
+  display: none;
+}
+
+.empty-tabs-menu{
+  display: none;
+}
+
 /* endregion Burger Menu Mode */
+#pucTabsMenuBar table td, .tabsSubMenuBar table td.gwt-MenuItem {
+  width: 130px;
+  display: inline-block;
+}
+
+#pucTabsMenuBar table td {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+.tabsSubMenuBar table td.gwt-MenuItem {
+  word-wrap:break-word;
+}
+
+.tabsSubMenuBar {
+  position: relative;
+}
+
+.tabsMenuContextSubMenu, .tabsSubMenuBar{
+  overflow-y: auto;
+}
+
+@media (max-width: 400px){
+  .tabsMenuContextSubMenu table td.gwt-MenuItem  {
+    width: calc(100vw - 235px);
+  }
+}
```

Comment:
Please, re-add the empty line at the end of the file (common best practice; you can configure IntelliJ to add it / not remove it on save).